### PR TITLE
FEDUP Token on Base

### PR DESCRIPTION
**Pull request by BalooSuperRareBears**
**Description**

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Review by wbnns — CHANGES_REQUESTED** [review]
@BalooSuperRareBears 

Hi there! This commit on this PR is failing CI / verification, could you please resolve?

**Comment by BalooSuperRareBears** [comment]
Hello, @wbnns 

I've synced my branch with the main one, let's see if it's ok now. Thanks!

File: data/FEDUP/data.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "FEDUP",
+  "symbol": "FEDUP",
+  "decimals": 4,
+  "description": "$FEDUP Clown Memetoken ,inspired by SRB  Bear Collection, now coming to BASE with legendary clown memes, real yield and DeFi.",
+  "website": "https://fedupclown.com",
+  "twitter": "https://twitter.com/Fedup_clown",
+  "tokens": {
+    "ethereum": {
+      "address": "0x6c80aeA8C83948D502c43539E22Ce2a2956Ad004"
```
Comment (wbnns) [inline data/FEDUP/data.json:10]: Hm, I don't see a token deployed at this address. Can you please confirm?

**Review by wbnns — CHANGES_REQUESTED** [review]
@BalooSuperRareBears 

Hello, we were unable to confirm the L1 token specified in your json. Can you please double-check?

**Comment by wbnns** [comment]
@BalooSuperRareBears 

Since it's been a while and we haven't heard back, closing this for now. Please let us know if you'd like to pursue the additional changes to resolve the outstanding issues on the PR and we can reopen. 👍

**Pull request closed without merging**